clusters/kind-cluster/charlie/dscp-node/release.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: charlie-node
+  namespace: charlie
+spec:
+  install:
+    remediation:
+      retries: -1
+  chart:
+    spec:
+      chart: dscp-node
+      version: "3.5.0"
+      sourceRef:
+        kind: HelmRepository
+        name: dscp-node
+  valuesFrom:
+  - kind: ConfigMap
+    name: charlie-values
+    valuesKey: values-dscp-node.yaml
+  interval: 10m0s
```

Comment:
not sure if this is possible but would be a lot easier to read if it was just `10m`